static/main.js

```diff
@@ -211,8 +218,8 @@ form.addEventListener("submit", (e) => {  // Submit button press event
     console_element.innerHTML = "";
 
     // Clear text field
-    let init_info = document.getElementById("init_info");
-    init_info.innerHTML = "";
+    let init_info_1 = document.getElementById("init_info_1");
+    init_info_1.innerHTML = "";
```

Comment:
Why do you clear innerHTML if on line 232 you assign this element a new value with "Processing started"